from0k2bp.tex

```diff
@@ -1294,8 +1333,8 @@
 include the critical recursion idea.
 
 The goal here is principally to reduce the amount of data communicated
-between the two parties (and when it's switched to a non-interactive
-form, using the Fiat Shamir heuristic, that means a more compact proof).
+between the two parties and when it's switched to a non-interactive
```

Comment:
Similar to a previous case, if you remove the parentheses you have to alter the punctuation - possibly using commas, and there are other ways. Probably just leave the parentheses.

Reply:
Reverted